### PR TITLE
Bump some resources for our tests of SO, for main

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

as per title

### CODE REVIEW

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml [hunk 1]
```diff
@@ -217,8 +217,8 @@ tests:
         limits:
           memory: 6Gi
         requests:
-          cpu: "2"
-          memory: 4Gi
+          cpu: "4"
+          memory: 6Gi
```
Comment (skonto) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml:221]: Have a pointer of the failure that leads to this adjustment (lacking context)?

Comment (matzew) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml:221]: we use same/similar setup on the kafka bits - that's why.

**Comment by matzew** [comment]
@skonto  is this a flake from serving:

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/27946/rehearse-27946-pull-ci-openshift-knative-serverless-operator-main-4.10-upstream-e2e-mesh-aws-ocp-410/1516413354841739264


/retest

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml (hunk 1 above)
Comment (skonto) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml:221]: Have a pointer for the Kafka bits? (lack context)

**Comment by skonto** [comment]
@matzew this job [has failed](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openshift-knative-serverless-operator-main-4.10-upstream-e2e-mesh-aws-ocp-410) some times lately and at least one time I saw that error [here](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1520/pull-ci-openshift-knative-serverless-operator-main-4.10-upstream-e2e-mesh-aws-ocp-410/1516487724645748736) in [PR](https://github.com/openshift-knative/serverless-operator/pull/1520). Eventually the test passed so it seems a flake.

**Comment by matzew** [comment]
@skonto so, should we try to tackle the flakes? 

For 4.9 all works, for 4.10... it seens flaky, hence this PR to give it a bit more room ... 

We can always revert the bump of resources 

Sounds good?

**Comment by skonto** [comment]
@matzew @nak3 has a PR for this https://github.com/openshift/knative-serving/pull/1102, backported a fix from upstream it was indeed a flake.

**Comment by skonto** [comment]
```
08:41:11.149 DEBUG:   [[ $(oc get -n kafka route.route.openshift.io/kafka-ui -ojsonpath='{.status.ingress[0].host}') == '' ]] : Waiting until non-zero (max 600 sec.)
```

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Pull request merged**